airbyte-integrations/bases/base-normalization/dbt-project-template/macros/incremental.sql

```diff
@@ -4,22 +4,22 @@
     - incremental_clause controls the predicate to filter on new data to process incrementally
 #}
 
-{% macro incremental_clause(col_emitted_at)  -%}
-  {{ adapter.dispatch('incremental_clause')(col_emitted_at) }}
+{% macro incremental_clause(col_emitted_at, tablename)  -%}
```

Comment:
if you add this extra argument of `tablename`, don't you have to add it to all the calls to this macro?

Reply:
nvm, i am guessing you are indeed changing the calls thanks to https://github.com/airbytehq/airbyte/pull/12846/files#r874512856

Reply:
I did mess this up though :P the lack of syntax highlighting actually makes it pretty hard to edit this file